packages/image/src/colorExtractor.ts

```diff
@@ -0,0 +1,78 @@
+interface PaletteOptions {
+  colorCount?: number;
+  quality?: number;
+}
+
+/**
+ * Extracts a palette of colors from a given image URL based on provided options.
+ * @param {string} imageUrl - The URL of the image.
+ * @param {PaletteOptions} [options] - The extraction options.
+ */
+export const colorExtractor = (imageUrl: string, options?: PaletteOptions) => {
```

Comment:
colorExtractor is not extractor, verb is better to name

```suggestion
export const colorExtractor = (imageUrl: string, options?: PaletteOptions) => {
```

Reply:
hmmmm.... How about `extractColors`?

Reply:
I agreed `extractColors` before in https://github.com/fepack/image/pull/33#discussion_r1333307262

Reply:
[e199d94](https://github.com/fepack/image/pull/33/commits/e199d94d96f06ab62623f04360b1b6865e91b7d9)